web/src/components/PoiFilters.tsx

```diff
@@ -49,11 +50,15 @@ const SortingHint = styled.div`
   padding: 0 4px;
 `
 
-const TileRow = styled.div<{ itemCount: number }>`
+const TileRow = styled(ToggleButtonGroup)<{ itemCount: number }>`
   display: grid;
   gap: 24px ${tileColumnGap}px;
   justify-content: center;
   grid-template-columns: repeat(${props => props.itemCount}, ${toggleButtonWidth}px);
+
+  & [class*='MuiToggleButton-root'] {
+    border-radius: 18px;
+  }
```

Comment:
🤔 I think we should avoid modifying mui styles in single components wherever possible and instead do that globally if necessary.

Reply:
This is not the default behavior so if I moved it to global it will affect also the feedback buttons.